packages/web-fetch/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import {reatomAsync} from '@reatom/async'
+import {Atom, Ctx, isAtom, throwReatomError} from '@reatom/core'
+
+export interface ReatomFetchOptions<O, I> {
+	fetch?: (url: string, init: RequestInit) => Promise<Response>
+	init?: Omit<RequestInit, 'method' | 'signal' | 'body' | 'headers'>
+	auth?: MaybeAtom<string>
+	headers?: MaybeAtom<HeadersInit>
+	baseUrl?: string
+	method?: string
```

Comment:
Adding things like `"get" | "post" | string` will enable code suggestions in some editors